### DISCUSSION TITLE
Image.open no such file or directory

### DISCUSSION THREAD

**Discussion by juliuseizure** [python-pillow/Pillow#7196] [Q&A] [answered]
I have been working with some starter code and have run into a snag.

I am using Image.open() to import one of the example images (cat.jpg in this case).  Instead I get FileNotFoundError: [Errno 2] No such file or directory: 'cat.jpg'

I dug into a different example I found online, and theirs gave a path to the image:  Image.open('data/src/lena.jpg').  Running that gives the same error.  In this case, yes, I found src (https://github.com/python-pillow/Pillow/tree/main/src), but that doesn't contain any images.  I did find some in https://github.com/python-pillow/Pillow/blob/main/Tests/images/.  Is this the new path we should be using?  EDIT:  trying Image.open("Tests/images/hopper.jpg") gives the same error.

**Comment by juliuseizure** [comment]
Resolved in an Issues post.

**Reply by hugovk** [reply, marked as answer]
For future reference: https://github.com/python-pillow/Pillow/issues/7197

**Reply by juliuseizure** [reply]
Yeah, that was me. For anyone as novice as me, just follow his link.
